packages/pluggableWidgets/slider-web/src/utils/createHandleGenerator.tsx

```diff
@@ -0,0 +1,50 @@
+import { createElement } from "react";
+import { SliderContainerProps } from "../../typings/SliderProps";
+import Tooltip from "rc-tooltip";
+import { Handle } from "rc-slider";
+
+// Copied from https://github.com/react-component/slider/blob/8.6.6/src/Slider.jsx#L165
+interface HandleGeneratorProps {
+    className: string;
+    prefixCls?: string;
+    vertical: boolean;
+    offset: number;
+    value: number;
+    dragging?: boolean;
+    disabled?: boolean;
+    min?: number;
+    max?: number;
+    index: number;
+    tabIndex?: number;
+    style?: React.CSSProperties;
+    ref?: React.Ref<any>;
+}
```

Comment:
Can we not make this `HandleProps` from `rc-slider` instead?

Reply:
Can you explain a bit more?

Reply:
WHat @Keraito means is to use the typings from `@types/rc-slider` instead of re-declare it (If it exists there)

Reply:
🤔 , I don't found any type in their library, this is why I wrote my one.

Reply:
Then lets just resolve this discussion. I was trying to check DefinitelyTyped but the files were removed for this lib in the new version.

Reply:
`import { Handle, HandleProps } from "rc-slider";`
![image](https://user-images.githubusercontent.com/5955441/142995117-a892ad09-1e80-457a-98b6-8862a0b532d8.png)

Reply:
@iobuhov @diego-antonelli